cmd/kubeadm/app/componentconfigs/defaults.go

```diff
@@ -107,15 +107,15 @@ func DefaultKubeletConfiguration(internalcfg *kubeadmapi.ClusterConfiguration) {
 	}
 
 	if externalkubeletcfg.StaticPodPath == "" {
-		externalkubeletcfg.StaticPodPath = kubeadmapiv1beta2.DefaultManifestsDir
-	} else if externalkubeletcfg.StaticPodPath != kubeadmapiv1beta2.DefaultManifestsDir {
-		warnDefaultComponentConfigValue(kind, "staticPodPath", kubeadmapiv1beta2.DefaultManifestsDir, externalkubeletcfg.StaticPodPath)
+		externalkubeletcfg.StaticPodPath = kubeadmapiv1beta3.DefaultManifestsDir
+	} else if externalkubeletcfg.StaticPodPath != kubeadmapiv1beta3.DefaultManifestsDir {
+		warnDefaultComponentConfigValue(kind, "staticPodPath", kubeadmapiv1beta3.DefaultManifestsDir, externalkubeletcfg.StaticPodPath)
```

Comment:
i think long term we can avoid this diff noise if we just name these imports implying internal and preferred external.
i think @timothysc had the same idea at some point.